lldb/test/API/commands/statistics/basic/main.cpp

```diff
@@ -0,0 +1,35 @@
+// Test that the lldb command `statistics` works.
```

Comment:
All this lock-talk makes me think we should have a test that exercises accessing the summary stats map from multiple threads. That way TSAN will help us in the test-suite.

Reply:
Sounds good, any sample test cases of multi threaded access I can steal the homework of?

Reply:
Not aware of anything like this off-the-top. We'd want a test that access the same `Target` from multiple threads and formats types. Maybe having some callback tied to several threads within the target which trigger a `ValueObject::GetSummaryAsCString` call. @clayborg @jimingham might have some more concrete ideas on how to do that.